spec/components/email/receiver_spec.rb

```diff
@@ -490,6 +496,19 @@ def process(email_name)
       expect(topic.topic_users.count).to eq(3)
     end
 
+    it "invites users with a secondary email in the chain" do
+      user1 = Fabricate(:user, trust_level: SiteSetting.email_in_min_trust)
+      Fabricate(:secondary_email, email: "discourse@bar.com", user: user1)
+      Fabricate(:secondary_email, email: "someone@else.com", user: user1)
+      user2 = Fabricate(:user, trust_level: SiteSetting.email_in_min_trust)
+      Fabricate(:secondary_email, email: "team@bar.com", user: user2)
+      Fabricate(:secondary_email, email: "wat@bar.com", user: user2)
+
+      expect { process(:cc) }.to change(Topic, :count)
+      user_ids = Topic.last.allowed_users.pluck(:id)
+      expect(user_ids).to contain_exactly(user1.id, user2.id)
```

Comment:
No need to pluck for the ids here since you can just do `expect(Topic.last.allowed_users).to contain_exactly(user1, user2)`